varnish/datadog_checks/varnish/varnish.py

```diff
@@ -326,7 +332,7 @@ def _parse_varnishadm(self, output):
             tokens = filter(None, line.strip().split(' '))
             tokens = [t for t in tokens]
             if len(tokens):
-                if tokens == ['Backend', 'name', 'Admin', 'Probe']:
+                if all(t in tokens for t in ['Backend', 'name', 'Admin', 'Probe']):
```

Comment:
It does. Instead of looking exactly for `['Backend', 'name', 'Admin', 'Probe']`, it will look for the same items in`['Backend', 'name', 'Admin', 'Probe', 'Health', 'Last', 'change']` (ignoring newly introduced columns) and skip the line.